source/_integrations/zha.markdown

```diff
@@ -10,6 +10,7 @@ ha_category:
   - Lock
   - Sensor
   - Switch
+  - Input Number
```

Comment:
We don't have a category for that, please remove it from this list.

Reply:
Done, thanks